app/models/game.rb

```diff
@@ -61,4 +62,18 @@ def location_is_under_attack_by_color?(color, row, col)
 
     false
   end
+
+  def forfeit!(user)
+    player_ids = [black_player_id, white_player_id]
+    unique_player_ids = player_ids.uniq
```

Comment:
If it's currently possible to have the same player as both `black` and `white`. We should add a validation to the model so that this never happens. Then you won't have to worry about this case here.